ports/servoshell/minibrowser.rs

```diff
@@ -189,64 +167,76 @@ impl Minibrowser {
                     ui.cursor().min.y
                 });
             *toolbar_height = Length::new(height);
-
+    
             CentralPanel::default()
                 .frame(Frame::none())
                 .show(ctx, |ui| {
                     let min = ui.cursor().min;
                     let size = ui.available_size();
                     let rect = egui::Rect::from_min_size(min, size);
                     ui.allocate_space(size);
-
```

Comment:
Try to run `./mach fmt` just before you make the commit, since it is failing the automatic checks because of formatting. You can use `./mach test-tidy` to verify that everything is fine before pushing.

Reply:
thanks for the input. 
```
munish@Zoro:~/servo$ ./mach fmt
munish@Zoro:~/servo$ ./mach test-tidy
 ➤  Checking config file (./servo-tidy.toml)...
 ➤  Checking directories for correct file extensions...
 ➤  Checking cargo lock (/home/munish/servo/Cargo.lock)...
 ➤  Checking files for tidiness
 ➤  Skipping WPT lint checks, because no relevant files changed.
 ➤  Checking formatting of rust files...
 ➤  Checking formatting of toml files...

 ✅ test-tidy reported no errors.
munish@Zoro:~/servo$
```